CHANGELOG.md

```diff
@@ -156,11 +156,14 @@
   <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>t</kbd>/<kbd>r</kbd>.
 - [Fixed a bug where selecting a nested breadcrumb would cause the order of
   breadcrumbs to change incorrectly.][6617]
+- [Cloud Dashboard, which supersedes the startup screen][6279]. Features also
+  added in various other PRs.
```

Comment:
This should be a user-friendly description. People form outside of the company should understand what changed, so such descriptions should be more wordy and provide more explanation of what really changed from user-perspective.